orc/src/main/java/org/apache/iceberg/orc/OrcIterable.java

```diff
@@ -0,0 +1,101 @@
+package org.apache.iceberg.orc;
```

Comment:
These files need the Apache license header.

Reply:
Done.